setup.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python
+from setuptools import setup
+import maxmindupdater
+
+with open('README.rst') as readme_file:
+    readme = readme_file.read().decode('utf8')
```

Comment:
That probably isn't what you want to do on python3.

I'd suggest using `from codecs import open` and using its `encoding` parameter. That'll give you a consistent behaviour across python 2 and 3.